api.swagger.yaml

```diff
@@ -1005,6 +1005,31 @@ paths:
               schema:
                 $ref: '#/components/schemas/uuid-list-schema'
       description: Search for groups. Returns list of uuids. Returns an empty array if no groups are found.
+    post:
+      tags:
+        - groups
+      responses:
+        '201':
+          description: The group was created successfully.
+          headers:
+            Location:
+              description: |
+                The URL at which the newly created group may be retrieved.
+              schema:
+                type: string
+        '400':
+          description: The group was not saved due to malformed JSON.
+        '422':
+          description: |
+            The group was not saved due to Drupal validation failure or JSON schema validation
+            failure.
+      description: Create a group.
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/group-schema'
```

Comment:
Some fields should be required on create

Reply:
updated